examples/z_put.py

```diff
@@ -76,29 +76,29 @@ def main():
     # --- Examples of put with other types:
 
     # - Integer
-    # session.put('/demo/example/Integer', 3)
+    # session.put('demo/example/Integer', 3)
 
     # - Float
-    # session.put('/demo/example/Float', 3.14)
+    # session.put('demo/example/Float', 3.14)
 
     # - Properties (as a Dictionary with str only)
-    # session.put('/demo/example/Properties', {'p1': 'v1', 'p2': 'v2'})
+    # session.put('demo/example/Properties', {'p1': 'v1', 'p2': 'v2'})
 
     # - Json (str format)
-    # session.put('/demo/example/Json',
-    #             json.dumps(['foo', {'bar': ('baz', None, 1.0, 2)}]),
+    # session.put('demo/example/Json',
+    #             json.dumps(['foo', {'bar': ('baz', None, 1.0, 2)}]).encode(),
     #             encoding=Encoding.TEXT_JSON)
```

Comment:
I found that it should be `Encoding.TEXT_JSON()` or it'll fail.

Reply:
You're right, that's something I don't understand the reason for...